clang/include/clang/AST/DeclarationName.h

```diff
@@ -682,6 +682,11 @@ class DeclarationNameTable {
   DeclarationName getCXXLiteralOperatorName(const IdentifierInfo *II);
 };
 
+struct CXXOperatorSourceInfo {
+  SourceLocation::UIntTy BeginOpNameLoc;
```

Comment:
I could have sworn I just saw this structure created/modified in another patch about a week ago?  Does this need a rebase?

Reply:
Interesting -- this patch was rebased recently on top of 0b6ddb02efdcbdac9426e8d857499ea0580303cd (from 4 days ago), and I don't recall any issues during the rebase.